### DISCUSSION TITLE
if the title contains a word in this format `word`

### DISCUSSION THREAD

**Discussion by dev-hanzo** [ObsidianPublisher/obsidian-github-publisher#83] [Q&A — Plugin]
![image](https://user-images.githubusercontent.com/117753674/213200806-b039d4bd-91d2-4c8a-8f64-cb7ff6a03c47.png)

I noticed that **if the title contains a word in this format `word`,** then the file will not be sent to github 

**[my set up](https://github.com/ObsidianPublisher/obsidian-github-publisher/discussions/67#discussion-4737298)**

**Comment by Mara-Li** [comment]
Frontmatter not valid. You need to add the `"` arround the title.
